internal/api/ui/login/external_provider_handler.go

```diff
@@ -780,3 +786,52 @@ func mapExternalNotFoundOptionFormDataToLoginUser(formData *externalNotFoundOpti
 		PreferredLanguage: language.Make(formData.Language),
 	}
 }
+
+func (l *Login) sessionParamsFromAuthRequest(ctx context.Context, authReq *domain.AuthRequest, identityProviderID string) ([]any, error) {
+	params := []any{authReq.AgentID}
+
+	if authReq.UserID != "" && identityProviderID != "" {
+		links, err := l.getUserLinks(ctx, authReq.UserID, identityProviderID)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
couldn't we just log the error and not return it? i guess it's better to just omit the hint than prevent the whole flow

Reply:
hm true, I would go a step further and ignore if an error occurs and continue the flow? I don't know if the login of the error gets you any additional information.